sdmetrics/visualization.py

```diff
@@ -32,7 +32,7 @@ def wrapper(*args, **kwargs):
             ipython_interpreter = str(get_ipython())
             if 'ZMQInteractiveShell' in ipython_interpreter and 'iframe' in renderers:
                 # This means we are using jupyter notebook
-                pio.renderers.default = 'vscode'
+                pio.renderers.default = 'iframe'
```

Comment:
I found that this change was accidentally added in a previous PR, so I reverted it.